Tests/AdyenTests/Adyen Tests/Components/Base/ApplePayDelegateMock.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ApplePayDelegateMock.swift
+//  AdyenUIKitTests
+//
+//  Created by Vladimir Abramichev on 02/05/2022.
+//  Copyright © 2022 Adyen. All rights reserved.
+//
+
+import Adyen
+import AdyenComponents
+import PassKit
+
+final class ApplePayDelegateMock: ApplePayComponentDelegate {
+
+    var didUpdate: ((PaymentComponentData, PaymentComponent) -> Void)?
+
+    func didUpdate(contact: PKContact, for payment: ApplePayPayment, with completion: @escaping (PKPaymentRequestShippingContactUpdate) -> Void) {
+        <#code#>
```

Comment:
placeholder forgotten I guess for below 3

Reply:
ups..
I deleted xcode reference, not file. Thanks!